src/lib/style/abstracts/_mixins.scss

```diff
@@ -142,13 +142,17 @@ $link-underline-color-default: adjust-color(
 // Core link styles across the portal main area.
 @mixin highlightLink() {
   color: SRC.$primary-action-color;
-  text-decoration: none;
   font-weight: 700;
   letter-spacing: 0.5px;
-  border-bottom: 1px solid $link-underline-color-default;
+  text-decoration: underline;
+  text-underline-offset: 4px;
+  text-decoration-thickness: 1px;
+  text-decoration-color: $link-underline-color-default;
   &:hover {
-    border-bottom: 2px solid map.get(SRC.$primary-color-palette, 700);
-    margin-bottom: -1px;
+    text-decoration: underline;
```

Comment:
Looks like this block is repeated, could you refactor and share (so we don't update one instance later)?